Svenbjorn/visitfaroeislands.com
# Added animation to menu display

Added animation to menu display.

TO TEST:
- Open and close the main menu, verify that it matches the animation from figma: https://www.figma.com/proto/eOshLuKuFSfE7gWDYgVcFb/VFI-Tourism---mobile?node-id=0-737&scaling=min-zoom&page-id=0%3A736&starting-point-node-id=0%3A737
